docs/_includes/fixture-wizard.html

```diff
@@ -0,0 +1,419 @@
+<svg width="854px" height="665px" viewBox="0 0 854 665" version="1.1">
```

Comment:
Fixed width, think will break widths on mobile.

Reply:
ah hmm. I don't know SVG very well but I'll see if I can it responsive.